travis/script/functions.sh

```diff
@@ -2,8 +2,9 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $SCRIPT_DIR/travis_functions.sh
 source $SCRIPT_DIR/predicate_functions.sh
 
-# idea taken from: http://blog.headius.com/2010/03/jruby-startup-time-tips.html
-export JRUBY_OPTS="${JRUBY_OPTS} -X-C" # disable JIT since these processes are so short lived
+# If JRUBY_OPTS isn't set, use these.
+# see http://docs.travis-ci.com/user/ci-environment/
+export JRUBY_OPTS=${JRUBY_OPTS:---server -Xcompile.invokedynamic=false"}
```

Comment:
It appears that you're change here is invalid bash, as the PRs I opened with your changes are all failing with a syntax error:

```
script/functions.sh: line 90: syntax error near unexpected token `)'
```

Any ideas how your change could have caused that?